### PR TITLE
[Scripts] update create script output

### PR DESCRIPTION
**Pull request by jacobsteves**
### WHY are these changes introduced?

Since we no longer change directories for the user, we need to update our output messages so that it is more clear. 

### WHAT is this pull request doing?

- Updates the output when creating a script to more accurately fit with the new flow.
- Followed the pattern of other projects

image

### CODE REVIEW

File: lib/project_types/script/layers/application/create_script.rb [hunk 1]
```diff
@@ -36,10 +37,10 @@ def install_dependencies(ctx, language, script_name, project, project_creator)
             ProjectDependencies.install(ctx: ctx, task_runner: task_runner)
           end
 
-          def bootstrap(ctx, project_creator)
+          def bootstrap(ctx, project, project_creator)
             UI::StrictSpinner.spin(ctx.message('script.create.creating')) do |spinner|
               project_creator.bootstrap
-              spinner.update_title(ctx.message('script.create.created'))
+              spinner.update_title(ctx.message('script.create.created', File.join(project.folder, project.source_file)))
```
Comment (SpencerHaan) [inline lib/project_types/script/layers/application/create_script.rb:43]: Does `project` not have a helper method on it that can do this already? It already has the information, and if it's `File` oriented already it makes sense. Could be added if it does not and would be useful? Like `project.source_path` or something.

File: lib/project_types/script/script_project.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ class ScriptProject < ShopifyCli::Project
     SOURCE_DIR = "src"
 
     attr_reader :extension_point_type, :script_name, :language
+    alias_method :folder, :script_name
```
Comment (SpencerHaan) [inline lib/project_types/script/script_project.rb:9]: What does this do?

File: lib/project_types/script/layers/application/create_script.rb (hunk 1 above)
Comment (jacobsteves) [reply lib/project_types/script/layers/application/create_script.rb]: Good idea 🙏 I added that

File: lib/project_types/script/script_project.rb (hunk 2 above)
Comment (jacobsteves) [reply lib/project_types/script/script_project.rb]: Makes `folder` an alias method of `script_name`. I was hoping to make an explicit connection between the folder name and script name, but I actually just ended up deleting this

**Comment by kwringe** [comment]
How about:

Change directories to july15 to run script commands.

**Comment by jacobsteves** [comment]
I was actually just copying how the other project types do it for consistency. Node and Rails both say `Change into your new project folder mynewfolder ...`.

But if you still prefer the other message, then I can change it!

**Comment by tanema** [comment]
@gfscott @attaboy for copy battles for changing directories 😆

**Comment by gfscott** [comment]
I'm with @kwringe on this one — let's go with the literal meaning of `cd` and make it `Change directories`. "Change into" sounds like swapping your outfit ("time to put on my script-editing poncho!")

**Comment by jacobsteves** [comment]
Great, thanks for the input!

**Comment by kwringe** [comment]
I'm glad that we're going with Change directories. But I'd also like to make some comments about this phrasing: _Change directories to your new project folder myfolder _

1) I would drop the word "new" from new project folder because I've always been taught that this is redundant. You've just created the project (and we told you this), so we don't need to emphasize the newness here.
2) I would also drop the word "folder" because I think that it's also a bit redundant/wordy here. If we really want to include this word, I would recommend that we replace folder with the word "directory" so that we are consistent. I prefer directory because I'd been taught that folder was a Microsoft term and directory was more inclusive (Linux)--though this should likely be challenged.

File: lib/project_types/script/layers/application/create_script.rb [hunk 3]
```diff
@@ -36,10 +37,10 @@ def install_dependencies(ctx, language, script_name, project, project_creator)
             ProjectDependencies.install(ctx: ctx, task_runner: task_runner)
           end
 
-          def bootstrap(ctx, project_creator)
+          def bootstrap(ctx, project, project_creator)
             UI::StrictSpinner.spin(ctx.message('script.create.creating')) do |spinner|
               project_creator.bootstrap
-              spinner.update_title(ctx.message('script.create.created'))
+              spinner.update_title(ctx.message('script.create.created', project.source_path))
```
Comment (ErinRenNumber1) [inline lib/project_types/script/layers/application/create_script.rb:43]: Nitpick: it seems that we only need to pass in `project.source_path` into this method, and not the entire `project`.  It's better to design the API to tell what it needs to know.  Now that I am looking at line 34, it feels a bit inconsistent, as `project.script_name` is the `script_name`, so really, you could only pass in `project`.  Alternatively, you can pass in `project.script_name` and `project.source_path` is probably better.

File: test/project_types/script/layers/application/create_script_test.rb [hunk 4]
```diff
@@ -19,7 +19,9 @@
   let(:task_runner) { stub(compiled_type: compiled_type) }
   let(:project_creator) { stub }
   let(:project_directory) { '/path' }
-  let(:script_project) { TestHelpers::FakeScriptProject.new(language: language, directory: project_directory) }
+  let(:script_project) do
+    TestHelpers::FakeScriptProject.new(language: language, directory: project_directory, script_name: script_name)
```
Comment (ErinRenNumber1) [inline test/project_types/script/layers/application/create_script_test.rb:23]: I feel I am missing something.  Why is this constructor taking in `directory`?

Comment (jacobsteves) [reply test/project_types/script/layers/application/create_script_test.rb:23]: Projects have a `directory` smart parameter

**Pull request merged**